gui/gui/src/TGTab.cxx

```diff
@@ -129,37 +130,65 @@ void TGTabElement::DrawBorder()
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Handle button event in the tab widget. Basically we only handle
-/// button events in the small tabs.
+/// button and scroll events in the small tabs.
 
 Bool_t TGTabElement::HandleButton(Event_t *event)
 {
-   if (event->fType == kButtonPress) {
-      TGTab* main = (TGTab*)fParent;
-      if (main) {
-         if (fShowClose && event->fWindow == GetId() &&
-             (UInt_t)event->fX > fTWidth+12 && (UInt_t)event->fX < fTWidth+26 &&
-             (UInt_t)event->fY > fHeight/2-7 && (UInt_t)event->fY < fHeight/2+7) {
-            if (main->GetTabTab(main->GetCurrent()) == this) {
-               main->CloseTab(main->GetCurrent()); // emit signal
-               //main->RemoveTab(main->GetCurrent());
-               return kTRUE;
+   if (event->fCode == kButton4 || event->fCode == kButton5) { //scroll wheel events
+      if (fParent) {
+         TGTab* main = (TGTab*)fParent;
+         #if defined(R__WIN32)
+         if (main->IsScrollingEnabled())
+         #else
+         if (main->IsScrollingEnabled() && event->fType == kButtonPress)
+         #endif
```

Comment:
I'll check if we can avoid this `#if defined(R__WIN32)`

Reply:
An alternative option would be to remove the kButtonRelease mask, then I would not need the ifdef. I checked and it works on Linux without the ifdef. But I do not know if that would break Windows.

Reply:
I commited some changes if you want to try those on Windows.